apiserver/common/crossmodel/interface.go

```diff
@@ -139,6 +139,14 @@ type Relation interface {
 
 	// SetSuspended sets the suspended status of the relation.
 	SetSuspended(bool, string) error
+
+	// ReplaceSettings replaces the application's settings within the
+	// relation.
+	ReplaceSettings(appName string, settings map[string]interface{}) error
```

Comment:
To be explicitly clear, can we name this ReplaceApplicationSettings
It also then fits with ApplicationSettings below

Reply:
Ok